client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -430,3 +434,23 @@ export const useCommandsArrayWpcom = ( {
 
 	return commands;
 };
+
+export const commandNameDisplayLabel: CommandNameDisplayLabel = {
```

Comment:
The display name for the each command is derived from whatever action is set in the callback so it ends up being something like `manageDNS` instead of `Manage DNS`. The easiest way to fix this is to edit the string set in the callback to be set to something like `Manage DNS` instead of `manageDNS` to make it more user-friendly. However, from searching around, it looks like this is not a good practice for naming callbacks. I went for some mapping here but happy to adjust the approach.

What are your thoughts @sejas ?

Reply:
Why not store this with the command definition itself?

Reply:
Yes, let's use a specific property in the command array.
We could set it when 

https://github.com/Automattic/wp-calypso/blob/91d0f444e26c67abf825e1b5f500113e5fb0968a/client/sites-dashboard/components/wpcom-smp-commands.tsx#L41-L46

Optionally, we may consider unifying those three variables (command name, placeholder, and breadcrumb name) under the same state.

Reply:
Thanks @sejas and @wojtekn - I did a refactor based on your suggestions and moved it to commands definitions